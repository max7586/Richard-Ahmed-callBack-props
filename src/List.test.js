import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import List from './List';

describe('<List />', () => {
  it ('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render( < List header = {'something'}
      cards = {[1, 2, 3]} key = {'5'} / > , div);

    //clean up
    ReactDOM.unmountComponentAtNode(div); 
  });
});

describe('<List />', () => {
  it('Renders this UI as expected', () => {
    const tree = renderer.create(< List header = {'something'}
      cards = {[1, 2, 3]} key = {'5'} / >).toJSON();
    expect(tree).toMatchSnapshot();
  });
});