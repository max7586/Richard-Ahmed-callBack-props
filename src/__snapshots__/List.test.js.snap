// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> Renders this UI as expected 1`] = `
<section
  className="List"
>
  <header>
    something
  </header>
  <div
    className="List-cards"
  >
    <div
      className="Card"
    >
      <h3 />
      <p />
    </div>
    <div
      className="Card"
    >
      <h3 />
      <p />
    </div>
    <div
      className="Card"
    >
      <h3 />
      <p />
    </div>
  </div>
</section>
`;
